bin/plugin/commands/performance.js

```diff
@@ -193,7 +175,11 @@ async function runTestSuite( testSuite, performanceTestDirectory ) {
  * @param {WPPerformanceCommandOptions} options  Command options.
  */
 async function runPerformanceTests( branches, options ) {
+	const branchesWereSpecified = branches?.length > 0;
 	const runningInCI = !! process.env.CI || !! options.ci;
+	const inferTestBranches = runningInCI && ! branchesWereSpecified;
+	const mergeRef = process.env.GITHUB_SHA ?? '';
+	const baseRef = process.env.GITHUB_BASE_REF ?? '';
```

Comment:
Personally I don't like that this PR ties the performance command with GitHub actions. I think a command to just pass a random number of branches/commits is way more understandable from an API perspective. 

Is it possible to leave this logic to figure out the right branch out of the command itself? potentially a new command?